tmt/beakerlib.py

```diff
@@ -167,6 +167,18 @@ def fetch(self):
         self.require = tmt.utils.listify(library.get('require', []))
         self.recommend = tmt.utils.listify(library.get('recommend', []))
 
+        # Create a symlink if the library is deep in the structure
+        if os.path.dirname(self.name).lstrip('/'):
+            link = self.name.lstrip('/')
+            path = os.path.join(self.tree.root, os.path.basename(self.name))
+            self.parent.debug(
+                f"Create a '{link}' symlink as the library is deep.")
```

Comment:
I would explicitely say `deep in directory structure`, just `deep` seems to vague

Reply:
-> `too deep in directory structure`

Otherwise it's just a statement, not an issue.

Reply:
Ok, I've extended the comment a bit.